components/dashboard/src/components/org-icon/styles.css

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+
+@layer components {
+    .bg-gradient-1 {
+        background: linear-gradient(45deg, theme('colors.gitpod-kumquat-dark') 25%, theme('colors.gitpod-kumquat') 75%);
```

Comment:
**suggestion**: We could probably drop and replace the following custom colors and drop all `gitpod-*` colors as most are not used anyhmore.


- `gitpod-kumquat-dark` ➡️ `orange-500`
- `gitpod-kumquat-light` ➡️ `orange-200`

Reply:
Oh gotcha - I'll swap these gradients to just use those directly, and then we can tackle deprecating the custom colors in our tailwind config.

Reply:
Updated—Resolving!